documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -15,69 +15,4 @@
 
 ### Resolved Issues
 
-* (IDETECT-3515) Resolved an issue where the Nuget Inspector was not supporting "<Version>" tags for "<PackageReference>" on the second line and was not cascading to Project Inspector in case of failure.
```

Comment:
@sig-cpotts We should keep this line still here as this is part of 9.2.0!!

Reply:
Absolutely, locally that line is still in the file... what the heck.

Reply:
@dmamidisynopsys Is it now appearing below?

Reply:
Oh wait my bad! there is change with "\ Ignore my message. Resolving this.